README.md

```diff
@@ -99,3 +99,62 @@ So as to specify right the target for scheme to run:
 The last step is to remove the red named folder `Removable Resources`.
 
 <img src="./resources/images/readme/scheme_remove-folder.png" width="500">
+
+</br>
+</br>
+</br>
+
+# Tuist Installation and Documentations
+
+</br>
+
+Run the following command in your terminal for the Tuist installation:
+
+</br>
+
+```bash
+bash <(curl -Ls https://install.tuist.io)
+```
+</br>
+
+Documentation : [Tuist Official Documents](https://docs.tuist.io/tutorial/get-started)
+
+</br>
+
+
+## How to use
+
+</br>
+
+- Copy the whole directory `Tuist` and its content to one directory up of your project's directory.
```

Comment:
We don't need to copy the whole directory `Tuist`. We will use this repository as a git template.